azkaban_cli/azkaban.py

```diff
@@ -266,6 +267,37 @@ def fetch_flows(self, project):
         logging.info('Project ID: %s' % (response_json[u'projectId']))
         return response_json
 
+    def fetch_jobs_from_flow(self, project, flow):
+        """Fetch jobs of a flow command, intended to make the request to Azkaban and return
+        the response.
+
+        This method receives the project name and flow id, makes the fetch jobs of a flow request
+        to fetch the jobs of a flow and evaluates the response.
+
+        If project, flow id is wrong or there is no session_id, it returns false.
+        If everything is fine, returns True.
```

Comment:
Doesn't look correct

Reply:
You're right. There are other functions with the same docstring issue. I will open an issue to correct all of them as well. Good catch!